docs/development/dependencies.md

```diff
@@ -256,6 +273,16 @@ from PyPI. (This is what makes poetry seem slow when doing the first
 `poetry install`.) Try `poetry cache list` and `poetry cache clear --all
 <name of cache>` to see if that fixes things.
 
+## Remove outdated egg-info
+
+Delete the `matrix_synapse.egg-info/` directory from the root of your Synapse
+install.
```

Comment:
Do we know (@erikjohnston maybe?) what causes this?

Reply:
I think `poetry install` does this now :/